packages/cfd/src/Containers/jurisdiction-modal-content.tsx

```diff
@@ -0,0 +1,832 @@
+import React from 'react';
+import { Icon, Text, Checkbox, Popover, StaticUrl } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
+import classNames from 'classnames';
+import { jurisdiction_contents } from '../Constants/jurisdiction-contents';
+import RootStore from '../Stores/index';
+import { connect } from '../Stores/connect';
+import { TExistingData } from '../Components/props.types';
+
+type TAvailableAccountAPI = [
+    {
+        market_type: string;
+        name: string;
+        requirements: {
+            signup: Array<string>;
+            withdrawal: Array<string>;
+        };
+        shortcode: string;
+        sub_account_type: string;
+    }
+];
+
+type TJurisdictionModalContent = {
+    account_type: string;
+    jurisdiction_selected_shortcode: string;
+    setJurisdictionSelectedShortcode: (card_type: string) => void;
+    synthetic_available_accounts: TAvailableAccountAPI;
+    financial_available_accounts: TAvailableAccountAPI;
+    poa_status: string;
+    poi_status: string;
+    is_eu: boolean;
+    context: RootStore;
+    is_fully_authenticated: boolean;
+    poi_poa_pending: boolean;
+    checked: boolean;
+    setChecked: React.Dispatch<React.SetStateAction<boolean>>;
+    real_synthetic_accounts_existing_data: TExistingData;
+    real_financial_accounts_existing_data: TExistingData;
+    poa_failed: boolean;
+    poi_failed: boolean;
+    is_virtual: boolean;
+    poi_verified_for_vanuatu: boolean;
+    poi_verified_for_labuan_bvi: boolean;
+    poa_verified: boolean;
+    poi_acknowledged_for_bvi_labuan: boolean;
+    need_poi_for_vanuatu: boolean;
+    need_poi_for_bvi_labuan: boolean;
+    poi_acknowledged_for_vanuatu: boolean;
+};
+
+type TJurisdictionCard = {
+    jurisdiction_selected_shortcode: string;
+    synthetic_available_accounts: TAvailableAccountAPI;
+    financial_available_accounts: TAvailableAccountAPI;
+    account_type: string;
+    poa_status: string;
+    poi_status: string;
+    poi_poa_none: boolean;
+    setJurisdictionSelectedShortcode: (card_type: string) => void;
+    type_of_card: string;
+    context: RootStore;
+    disabled: boolean;
+    poa_failed: boolean;
+    poi_failed: boolean;
+    poa_acknowledged: boolean;
+    poi_acknowledged: boolean;
+    is_fully_authenticated: boolean;
+    is_virtual: boolean;
+    poi_verified_for_vanuatu: boolean;
+    poi_verified_for_labuan_bvi: boolean;
+    poa_verified: boolean;
+    poi_acknowledged_for_bvi_labuan: boolean;
+    need_poi_for_vanuatu: boolean;
+    need_poi_for_bvi_labuan: boolean;
+    poi_acknowledged_for_vanuatu: boolean;
+};
+const StatusCodes = {
+    none: 'none',
+    pending: 'pending',
+    rejected: 'rejected',
+    verified: 'verified',
+    expired: 'expired',
+    suspected: 'suspected',
+};
```

Comment:
```suggestion
enum StatusCodes {
    None = 'none',
    Pending = 'pending',
    Rejected = 'rejected',
    Verified = 'verified',
    Expired = 'expired',
    Suspected = 'suspected',
};
```
The way the object is declared looks like it can be an enum

Reply:
I think there is an issue with enum because if i use it, it says i already declared it.